tasks/connectors/bitsight/bitsight.rb

```diff
@@ -28,6 +28,11 @@ def self.metadata
               required: false,
               default: true,
               description: "Create (informational) vulns for findings labeled benign" },
+            { name: "bitsight_company_guids",
+              type: "string",
+              required: false,
+              default: "",
+              description: "Comma separated list of company guids to use for data pull. If nil, script will pull for 'My Company' only" },
```

Comment:
Is 'guids', 'GUIDs' or something else?

Reply:
I'm going to go ahead and merge. Grab this on next time.